src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="logo"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="header_logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav__ul">
+          <li class="header__nav__ul__li">
+            <a
+              href="Apple"
+              class="header__nav__ul__li__a is-active"
+            >
```

Comment:
apply these recommendations to the code below as well
```suggestion
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
            >
```